packages/SwingSet/src/controller/initializeSwingset.js

```diff
@@ -172,8 +173,8 @@ export function loadBasedir(basedir, options = {}) {
  *    determined.
  */
 async function resolveSpecFromConfig(referrer, specPath) {
+  await null;
```

Comment:
why is this a problem,
```js
return new URL(await resolveModuleSpecifier(specPath, referrer)).pathname;
```
and this is not?
```js
  const url = await resolveModuleSpecifier(specPath, referrer);
  return new URL(url).pathname;
```

Reply:
Visibility and reviewability of the `await` boundary. If the separating await is on an argument expression:

```js
const foo = bar(f(), await g(), h());
```

First and most important, it is too easy to miss. We could require a noisy comment, but it is easier and more pleasant to just require that the `await` expression be extracted to top level of the function body.

Second: This notation makes it somewhat harder to understand what effects happen before the await (`f()` and `g()`) and what after the await (`h()` and `bar)...)`). The fact that the await happens *between* `g()` and `h()` is especially surprising and easy to miss if you're reading code in a hurry.